src/global-firestore-data/index.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Put all functions that can use and change the firestore data here.
+ */
+import store from '@/store';
+
+import { cornellCourseRosterCourseToFirebaseSemesterCourse } from '@/user-data-converter';
+import { incrementUniqueID } from './utils';
+
+export const cornellCourseRosterCourseToFirebaseSemesterCourseWithGlobalData = (
+  course: CornellCourseRosterCourse
+): FirestoreSemesterCourse =>
+  cornellCourseRosterCourseToFirebaseSemesterCourse(course, store, incrementUniqueID);
+
+export { incrementUniqueID };
```

Comment:
Left the re-export as `import ... from ...` then `export ...` rather than `export ... from ...` because `incrementUniqueID` is used in `cornellCourseRosterCourseToFirebaseSemesterCourseWithGlobalData`.

Not sure if this should go in `index.ts`, `utils.ts`, or its own dedicated file. I figured it could go in `index.ts` because it's basically the same function as the one in `@/user-data-converter`, but I also think moving it elsewhere would be reasonable (eg., if we want to keep `index.ts` re-exports only).

Reply:
Why can't `cornellCourseRosterCourseToFirebaseSemesterCourseWithGlobalData` go in `user-data-converter`?

Reply:
I think leaving `cornellCourseRosterCourseToFirebaseSemesterCourseWithGlobalData` here for now is fine. If we end up creating more functions of the same type we should put them in a separate file, but for now it doesn't look too messy.